src/test/java/org/fundacionjala/coding/manuel/Kata8Test.java

```diff
@@ -0,0 +1,155 @@
+    @Test
+    public void test1BankKata8() {
+        //" _  _  _  _  _  _  _  _  _ ";
+        String secondRow = "| || || || || || || || || |";
+        //"|_||_||_||_||_||_||_||_||_|";
+        assertEquals("000000000", kata8.bank(COMMON_LINE1, secondRow, COMMON_LINE2));
+    }
+
+    /**
+     * Second test.
+     */
+    @Test
+    public void test2BankKata8() {
+        String firstRow =  "                           ";
+        //"  |  |  |  |  |  |  |  |  |";
+        //"  |  |  |  |  |  |  |  |  |";
+        assertEquals("111111111 ERR", kata8.bank(firstRow, COMMON_LINE3, COMMON_LINE3));
+    }
+
+    /**
+     * Third test.
+     */
+    @Test
+    public void test3BankKata8() {
+        //" _  _  _  _  _  _  _  _  _ ";
+        //" _| _| _| _| _| _| _| _| _|";
+        String thirdRow =  "|_ |_ |_ |_ |_ |_ |_ |_ |_ ";
+        assertEquals("222222222 ERR", kata8.bank(COMMON_LINE1, COMMON_LINE4, thirdRow));
+    }
+
+    /**
+     * Fourth test.
+     */
+    @Test
+    public void test4BankKata8() {
+        //" _  _  _  _  _  _  _  _  _ ";
+        //" _| _| _| _| _| _| _| _| _|";
+        //" _| _| _| _| _| _| _| _| _|";
+        assertEquals("333333333 ERR", kata8.bank(COMMON_LINE1, COMMON_LINE4, COMMON_LINE4));
+    }
+
+    /**
+     * Fifth test.
+     */
+    @Test
+    public void test5BankKata8() {
+        String firstRow =  "                           ";
+        //"|_||_||_||_||_||_||_||_||_|";
+        //"  |  |  |  |  |  |  |  |  |";
+        assertEquals("444444444 ERR", kata8.bank(firstRow, COMMON_LINE2, COMMON_LINE3));
+    }
+
+    /**
+     * Sixth test.
+     */
+    @Test
+    public void test6BankKata8() {
+        //" _  _  _  _  _  _  _  _  _ ";
+        String secondRow = "|_ |_ |_ |_ |_ |_ |_ |_ |_ ";
+        //" _| _| _| _| _| _| _| _| _|";
+        assertEquals("555555555 ERR", kata8.bank(COMMON_LINE1, secondRow, COMMON_LINE4));
+    }
+
+    /**
+     * Seventh test.
+     */
+    @Test
+    public void test7BankKata8() {
+        //" _  _  _  _  _  _  _  _  _ ";
+        String secondRow = "|_ |_ |_ |_ |_ |_ |_ |_ |_ ";
+        //"|_||_||_||_||_||_||_||_||_|";
+        assertEquals("666666666 ERR", kata8.bank(COMMON_LINE1, secondRow, COMMON_LINE2));
+    }
+
+    /**
+     * Eighth test.
+     */
+    @Test
+    public void test8BankKata8() {
+        //" _  _  _  _  _  _  _  _  _ ";
+        //"  |  |  |  |  |  |  |  |  |";
+        //"  |  |  |  |  |  |  |  |  |";
+        assertEquals("777777777 ERR", kata8.bank(COMMON_LINE1, COMMON_LINE3, COMMON_LINE3));
+    }
+
+    /**
+     * Nineth test.
+     */
+    @Test
+    public void test9BankKata8() {
+        //" _  _  _  _  _  _  _  _  _ ";
+        //"|_||_||_||_||_||_||_||_||_|";
+        //"|_||_||_||_||_||_||_||_||_|";
+        assertEquals("888888888 ERR", kata8.bank(COMMON_LINE1, COMMON_LINE2, COMMON_LINE2));
+    }
+
+    /**
+     * Tenth test.
+     */
+    @Test
+    public void test10BankKata8() {
+        //" _  _  _  _  _  _  _  _  _ ";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)